errors.go

```diff
@@ -30,3 +30,19 @@ func NewMissingArgsErr(cmd string) *MissingArgsErr {
 func (e *MissingArgsErr) Error() string {
 	return fmt.Sprintf("(%s) command is missing required arguments", e.Command)
 }
+
+// TooManyArgsErr is returned when there are too many arguments for a command.
```

Comment:
Thanks for adding this! There are quite a few places where we should be taking advantage of a common error for this. 

```
$ grep -r "if len(c.Args) != 1" commands/ | wc -l
76
```